### ISSUE TITLE
Add support for Wallet Connect

### ISSUE THREAD

**Issue by vikmeup** [TrustWallet/trust-wallet-ios#768]

**Comment by JamesANZ** [comment]
I noticed this is in the latest trust app yet I cannot find the implementation source code in this repo, could you point me in the right direction please? @vikmeup

**Comment by vikmeup** [comment]
https://github.com/WalletConnect/swift-walletconnect-lib

**Comment by JamesANZ** [comment]
@vikmeup yes but that is the pod repo, I mean the implementation code inside the iOS app itself.

**Comment by vikmeup** [comment]
You would need to implement it yourself to fit your architecture

**Comment by JamesANZ** [comment]
Yes I know but what I mean is, where is the implementation code you use for Trust iOS?
